CONTRIBUTING.md

```diff
@@ -80,6 +80,31 @@ Now you will be automatically reminded to add docstrings to your code. `black` w
 
 Run `git config blame.ignoreRevsFile .github/.git-blame-ignore-revs`
 
+## Code style conventions:
+
+Most of our codebase is written in Python. 
+
+
+### Python version
+We support all the Python version starting with and above 3.7. It means that we require in every contribution that the 
+code is compatible with python 3.7, and therefore we can't accept contribution that used features from the newest python versions.
+
+### Code formatting
+
+All of our Python codebase follow formatting standard. We require that every code contribution is formatted using [black](https://github.com/psf/black)
```

Comment:
if we're refferring to pep8 standard let's mention it (because that's what black does)

Reply:
u are right

Reply:
Black follow pep8 but go above and do some custom things. I can add that we follow pep8 but we still need to say that we follow black conventions